setup.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+
+from setuptools import find_packages, setup
+
+package_name = 'openslides-export'
+module_name = 'openslides_export'
+
+# The following commented unique string is used to detect this import.
+module = __import__(module_name)  # EuC5Kee6ohp7aesae3yai3fe3soo2uzoshohk2uj
```

Comment:
The string should be unique and not the same as in other plugins to be sure the the import restriction actually works well.